syntax/table.php

```diff
@@ -63,7 +63,17 @@ public function handle($match, $state, $pos, Doku_Handler $handler){
 
         try {
             $parser = new ConfigParser($lines);
-            return  $parser->getConfig();
+            $config = $parser->getConfig();
+            /** @var helper_plugin_struct $struct_helper */
+            $struct_helper = plugin_load('helper', 'struct');
+            foreach ($config['schemas'] as $schema) {
+                $result = $struct_helper->getSchema($schema[0]);
+                if (!$result[$schema[0]]->getId()) {
+                    msg(sprintf($this->getLang('error:schema missing'), $schema[0]),-1);
+                    return false;
```

Comment:
we could simply throw a new exception here and thus have all error handling a few lines below.